Xamarin.Forms.Core/AutoCapitalization.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Xamarin.Forms
+{
+    public enum AutoCapitalization
```

Comment:
doesn't it make sense to capitalize it as `Autocapitalization` ? like it is in iOS ?